src/command_modules/azure-cli-monitor/setup.py

```diff
@@ -0,0 +1,55 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup
+
+VERSION = '0.1.1b2+dev'
+
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'License :: OSI Approved :: MIT License',
+]
+
+DEPENDENCIES = [
+    'azure-cli-core',
+    'azure-monitor==0.1.0'
+    # 'azure-mgmt-monitor==0.1.0'
```

Comment:
We have allowed services (KeyVault, IoT, etc.) to merge PRs with embedded clients. DocumentDB was about to earlier this week but the SDK was released so he went ahead and did the import changes.

If something holds up this PR from being merged, it really doesn't need to be the desire to avoid (temporarily) having the SDK code in the module.